From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mark Salyzyn <salyzyn@google.com>
Date: Tue, 3 Sep 2019 09:42:35 -0700
Subject: ANDROID: GKI: enable CONFIG_SPI for x86

Adds compile coverage for SPI in the TH builds, runtime no so much.

Signed-off-by: Mark Salyzyn <salyzyn@google.com>
Bug: 140290328
Change-Id: I2d7777ab0e671248084880e5c1770b6ec6d7a650
---
 arch/x86/configs/gki_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index e86cd1009a67..9b76d55e8c41 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -240,6 +240,7 @@ CONFIG_HW_RANDOM=y
 CONFIG_HW_RANDOM_VIRTIO=y
 # CONFIG_I2C_COMPAT is not set
 # CONFIG_I2C_HELPER_AUTO is not set
+CONFIG_SPI=y
 CONFIG_GPIOLIB=y
 # CONFIG_HWMON is not set
 CONFIG_DEVFREQ_THERMAL=y
